code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -205,6 +206,7 @@ Borg Shaker
 	charge_cost = 20 //Lots of reagents all regenerating at once, so the charge cost is lower. They also regenerate faster.
 	recharge_time = 3
 	accepts_reagent_upgrades = FALSE
+	dispensed_temperature = 274.5
```

Comment:
Where does this number come from? Ideally, this would either be dynamically created or derived from some define (like T0C).

Reply:
The value is what I came up with after looking at /datum/chemical_reaction/ice and /datum/chemical_reaction/water, both of which are in others.dm and start at line 733. The required_temp for ice is 274, and the required_temp for water is 275. By choosing a temperature between the two, neither reaction occurs. I have tested this and confirmed that both ice and water, as well as any other reagents from the borg shaker will not perform the ice or water reaction when poured into a container.
As for using a define- In #57114, which I heavily referenced, they used "T0C + 0.85" which gives us exactly 274. I tried this, it was freezing the water as it was being produced by the shaker. I could use "T0C + 1.35" which would solve to 274.5K as well, but I thought just using 274.5 in the first place was better. I can change it to T0C + 1.35 if you want.

Reply:
`T0C + 1.35` is better, though the best would be automatically deriving this. As in, dynamically getting the temperature of ice and water, and choosing the middle between that (and caching it).

Reply:
Makes sense, but I'm not sure how to reference those values since they are private to those datums. I think. Do you have any examples elsewhere in the code I could look at?

Reply:
I changed it to T0C + 1.35. Also did the emag module for consistency and to prevent identification in an edge case of fake beer mixed in a drink containing ice. I tested it again locally to make sure they both work.